gobblin-runtime/src/main/java/org/apache/gobblin/runtime/util/MultiWorkUnitUnpackingIterator.java

```diff
@@ -31,29 +31,59 @@
 @RequiredArgsConstructor
 public class MultiWorkUnitUnpackingIterator implements Iterator<WorkUnit> {
   private final Iterator<WorkUnit> workUnits;
+
+  /** The iterator for {@link #nextWu} if it's a {@link MultiWorkUnit} */
```

Comment:
Why is this file being changed?

Reply:
This is to fix a bug where the iterator has tailing `MultiWorkUnit` with empty work unit list.